docs/videos.md

```diff
@@ -49,6 +49,14 @@ Check your video, note it may be splitted in many files if is too long
 
 ## Customizations
 
+### Audio
+If you want to additionaly record audio with the video use this
+
+    docker run --rm --name=grid -p 4444:24444 -p 5920:25900 \
+      --shm-size=1g -e VNC_PASSWORD=hola \
```

Comment:
selenium suggests 2g instead of 1g https://github.com/SeleniumHQ/docker-selenium/#running-the-images OR /dev/shm

Reply:
Just copied from the other example above in RUN. I can change that tho.